src/3_branch.js

```diff
@@ -299,14 +311,37 @@ Branch.prototype['init'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
 		// Keep android titanium from calling close
 		if (self.keepAlive) { setTimeout(function() { self.keepAlive = false; }, 2000); }
 		done(err, data && utils.whiteListSessionData(data));
+	},
+
+	attachVisibilityEvent = function() {
+		var hidden, changeEvent;
+		if (typeof document.hidden != undefined) {
+			hidden = "hidden",
+			changeEvent = "visibilitychange";
+		}
+		else if (typeof document.mozHidden != undefined) {
+			hidden = "mozHidden",
+			changeEvent = "mozvisibilitychange";
+		}
+		else if (typeof document.msHidden != undefined) {
+			hidden = "msHidden",
+			changeEvent = "msvisibilitychange";
+		}
+		else if (typeof document.webkitvisibilitychange != undefined) {
```

Comment:
Ah good catch. The 1st case of document.hidden probably worked since I have an up to dat safari